src/en/hentairead/src/eu/kanade/tachiyomi/extension/en/hentairead/Hentairead.kt

```diff
@@ -20,6 +21,23 @@ class Hentairead : Madara("HentaiRead", "https://hentairead.com", "en", dateForm
 
     override val versionId: Int = 2
 
+    private val cdnDomain = "https://hentaimanga.to"
+
+    private val cdnHeaders = super.headersBuilder()
+        .add("Accept", "image/avif,image/webp,image/apng,image/svg+xml,image/*,*/*;q=0.8")
+        .build()
+
+    override val client = super.client.newBuilder()
+        .addInterceptor { chain ->
+            val request = chain.request()
+            val url = request.url.toString()
+            if (url.startsWith("$cdnDomain/wp-content/uploads/")) {
```

Comment:
domain can be inlined, but is it necessary to put domain here? just checking `/wp-content/uploads/` should be fine no?